gateway-service/pom.xml

```diff
@@ -24,10 +24,20 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+            <version>2.17.2</version>
```

Comment:
Please extract the version in the parent pom's dependency management.
I have a feeling this will be used by multiple modules